src/crypto.rs

```diff
@@ -86,6 +86,20 @@ pub fn fast_aggregate_verify(pks: &[&PublicKey], msg: &[u8], signature: &Signatu
     res == BLST_ERROR::BLST_SUCCESS
 }
 
+pub fn eth_aggregate_public_keys<const N: usize>(
+    pks: &Vector<PublicKey, N>,
```

Comment:
```suggestion
    pks: &[PublicKey],
```

and then we can drop the const generic param here

I think here we are fine to just take a slice of keys, it may mean callers get a bit more complex but that's ok imo